cmd/mt-http-gw/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/grafana/metrictank/logger"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"runtime"
+)
+
+var (
+	version       = "(none)"
+	showVersion   = flag.Bool("version", false, "print version string")
+	metrictankUrl = flag.String("metrictank-url", "", "the url of the metrictank instance to proxy requests to")
+	bindAddress   = flag.String("bind-address", ":8080", "the address to bind to")
```

Comment:
thoughts on a default port?

Reply:
Use the same default that is in tsdb-gw

Reply:
oh right, duh... 0784c9581d1da39dd1190e4969fbabc9efa8cf29